v2/skins.js

```diff
@@ -80,6 +84,7 @@
 //   * "ShowInWardrobe"
 //   * "HideIfLocked" -- doesn't show up in the wardrobe if not unlocked
 //   * "NoCost" -- doesn't cost transmutation charges to apply
+//   * "OverrideRarity" -- skin overrides item rarity when applied
```

Comment:
e.g., if you apply a legendary skin to an item, it changes the item's border to legendary.

Pretty sure that's what the flag does.